packages/@lwc/ssr-runtime/src/lightning-element.ts

```diff
@@ -29,31 +29,33 @@ interface PropsAvailableAtConstruction {
     tagName: string;
 }
 
+export const SYMBOL__SET_INTERNALS = Symbol();
```

Comment:
```suggestion
export const SYMBOL__SET_INTERNALS = Symbol('set-internals');
```

To aid in debuggability.